helm_deploy/cmd-api/values.yaml

```diff
@@ -57,14 +57,12 @@ generic-service:
   namespace_secrets:
     cmd-api:
       APPINSIGHTS_INSTRUMENTATIONKEY: "APPINSIGHTS_INSTRUMENTATIONKEY"
+      OAUTH_CLIENT_ID: "API_CLIENT_ID"
+      OAUTH_CLIENT_SECRET: "API_CLIENT_SECRET"
 
     check-my-diary-notifications:
       APPLICATION_NOTIFY_KEY: NOTIFY_CLIENT_KEY
 
-    check-my-diary:
```

Comment:
was sharing with check my diary - now has own storage and different client values